client/packages/system/src/Name/ListView/ImportProperties/PropertiesImportModal.tsx

```diff
@@ -0,0 +1,239 @@
+      const remainingRecords = bufferedFacilityProperties;
+      while (remainingRecords.length) {
+        await Promise.all(
+          remainingRecords.splice(0, 10).map(async facility => {
+            await mutateAsync(toUpdateNamePropertiesInput(facility))
+              .then(async result => {
+                // Map structured Errors
+                if (result?.__typename === 'Mutations') {
+                  // const errorMessage = mapStructuredErrors(result);
+                  const errorMessage = result;
+                  importErrorRows.push({
+                    ...facility,
+                    errorMessage: errorMessage.__typename,
+                  });
+                  return;
+                }
+              })
+              .catch((err: { message: string }) => {
+                if (!err) {
+                  err = { message: t('messages.unknown-error') };
+                }
+                importErrorRows.push({
+                  ...facility,
+                  errorMessage: err.message,
+                });
+              });
+          })
+        ).then(() => {
+          // Update Progress Bar
+          const percentComplete =
+            10 - (remainingRecords.length / numberImportRecords) * 100.0;
+          setImportProgress(percentComplete);
+          setImportErrorCount(importErrorRows.length);
+        });
+      }
+      if (importErrorRows.length === 0) {
+        const importMessage = t('messages.import-generic', {
+          count: numberImportRecords,
+        });
+        const successSnack = success(importMessage);
+        successSnack();
+        onChangeTab(Tabs.Upload);
+        setBufferedFacilityProperties([]);
+        setErrorMessage('');
+        onClose();
+      } else {
+        // Load the error rows in to the component for review
+        setErrorMessage(t('messages.import-generic'));
+        setBufferedFacilityProperties(importErrorRows);
+        setImportErrorCount(importErrorRows.length);
+        onChangeTab(Tabs.Review);
+      }
+    }
+  };
+
+  const onClickStep = (tabName: string) => {
+    switch (tabName) {
+      case Tabs.Upload:
+        changeTab(tabName as Tabs);
+        break;
+      case Tabs.Review:
+        changeTab(tabName as Tabs);
+        break;
+      case Tabs.Import:
+        // Do nothing, user can't get to the import page without clicking the import button
+        break;
+    }
+  };
+
+  const changeTab = (tabName: Tabs) => {
+    switch (tabName) {
+      case Tabs.Upload:
+        setErrorMessage('');
+        setActiveStep(0);
+        break;
+      case Tabs.Review:
+        setActiveStep(1);
+        break;
+      case Tabs.Import:
+        setActiveStep(2);
+        break;
+    }
+    onChangeTab(tabName);
+  };
+
+  const showWarnings = errorMessage.length == 0 && warningMessage.length > 0;
+
+  console.info('showWarnings', showWarnings);
+
+  const importSteps = [
+    { label: t('label.upload'), description: '', clickable: true },
```

Comment:
<img width="1241" alt="Screenshot 2024-06-25 at 9 59 33 AM" src="https://github.com/msupply-foundation/open-msupply/assets/55115239/73181527-7b2f-41d0-9e84-e9a569db03b4">

Translations for the modal aren't coming through, same for the upload error message... also title should be import facility properties rather than CCE :grin: